Doc/whatsnew/3.11.rst

```diff
@@ -1070,7 +1071,8 @@ Deprecated
   * :mod:`spwd`
   * :mod:`sunau`
 
-  (Contributed by Brett Cannon in :issue:`47061`.)
+  (Contributed by Brett Cannon in :issue:`47061` and Victor Stinner in
+  :gh:`68966`.)
```

Comment:
Looking at the others, we should include these `3.11.rst` changes in the 2nd PR that'll do the code warnings, because we don't want to backport this bit.

Reply:
I prefer to modify the backport but push all changes at once in the main branch.